code/modules/client/edge_sliding/move_loop.dm

```diff
@@ -61,10 +61,10 @@
 	set waitfor = 0
 	if(src.mloop) return
 	mloop = 1
-	src.Move(get_step(mob.loc, true_dir),true_dir)
-	while(src.true_dir)
+	Move(get_step(mob.loc, true_dir),true_dir)
+	while(true_dir)
 		sleep(world.tick_lag)
```

Comment:
another missed sleep

Reply:
I cant remove that sleep, going to port ssmovement form tg for that